api/organisations/tests/test_views.py

```diff
@@ -581,42 +583,132 @@ def setUp(self) -> None:
         self.subscription = Subscription.objects.get(organisation=self.organisation)
 
     @mock.patch("organisations.models.get_plan_meta_data")
-    def test_when_subscription_plan_is_changed_max_seats_and_max_api_calls_are_updated(
-        self, mock_get_plan_meta_data
-    ):
-        # Given
-        new_plan_id = "new-plan-id"
-        new_max_seats = 3
-        new_max_api_calls = 100
+    @mock.patch("organisations.views.extract_subscription_metadata")
+    def mocked_subscription_plan_data(
+        self,
+        mock_get_subscription_metadata: ChargebeeObjMetadata,
+        mock_get_plan_meta_data: dict[str, str],
```

Comment:
These types are not correct - these objects are `MagicMock`s.

Reply:
Corrected.